go_sequencer/common/batch.go

```diff
@@ -8,37 +8,35 @@ import (
 	ethCommon "github.com/ethereum/go-ethereum/common"
 )
 
-const batchNumBytesLen = 8
+const batchNumBytesLen = 8 //TODO: Need to check if this needs to be updated
 
 // Batch is a struct that represents Hermez network batch
 type Batch struct {
 	BatchNum  BatchNum       `meddler:"batch_num"`
 	EthTxHash ethCommon.Hash `meddler:"eth_tx_hash"`
 	// Ethereum block in which the batch is forged
-	EthBlockNum        int64                `meddler:"eth_block_num"`
-	ForgerAddr         ethCommon.Address    `meddler:"forger_addr"`
-	CollectedFees      map[TokenID]*big.Int `meddler:"fees_collected,json"`
-	FeeIdxsCoordinator []AccountIdx         `meddler:"fee_idxs_coordinator,json"`
-	StateRoot          *big.Int             `meddler:"state_root,bigint"`
-	NumAccounts        int                  `meddler:"num_accounts"`
-	LastIdx            int64                `meddler:"last_idx"`
-	ExitRoot           *big.Int             `meddler:"exit_root,bigint"`
-	GasUsed            uint64               `meddler:"gas_used"`
-	GasPrice           *big.Int             `meddler:"gas_price,bigint"`
-	EtherPriceUSD      float64              `meddler:"ether_price_usd"`
+	EthBlockNum   int64             `meddler:"eth_block_num"`
+	ForgerAddr    ethCommon.Address `meddler:"forger_addr"`
+	StateRoot     *big.Int          `meddler:"state_root,bigint"`
+	NumAccounts   int               `meddler:"num_accounts"`
+	LastIdx       int64             `meddler:"last_idx"`
+	ExitRoot      *big.Int          `meddler:"exit_root,bigint"`
+	GasUsed       uint64            `meddler:"gas_used"`
+	GasPrice      *big.Int          `meddler:"gas_price,bigint"`
+	EtherPriceUSD float64           `meddler:"ether_price_usd"`
 	// ForgeL1TxsNum is optional, Only when the batch forges L1 txs. Identifier that corresponds
 	// to the group of L1 txs forged in the current batch.
 	ForgeL1TxsNum *int64   `meddler:"forge_l1_txs_num"`
 	SlotNum       int64    `meddler:"slot_num"` // Slot in which the batch is forged
 	TotalFeesUSD  *float64 `meddler:"total_fees_usd"`
 }
 
-type BatchNum int64
+type BatchNum uint32
```

Comment:
I'm curious why we are changing this. Is the assumption we won't reach `4294967295` batch number, which is the largest number `uint32` can hold?

Reply:
It's on the basis of the struct doc shared here - [Link](https://www.notion.so/tokamak/Rollup-Circuit-Specification-fa2729166fb34a91b5fe8dcd4b0dcf40)